tests/performance/lower_upper_function.xml

```diff
@@ -1,11 +1,11 @@
 <test>
-    <query>select lower(randomString(16))</query>
-    <query>select lower(randomString(32))</query>
-    <query>select lower(randomString(64))</query>
-    <query>select lower(randomString(128))</query>
-    <query>select lower(randomString(256))</query>
-    <query>select lower(randomString(512))</query>
-    <query>select lower(randomString(1024))</query>
-    <query>select lower(randomString(832))</query>
-    <query>select lower(randomString(416))</query>
+    <query>select lower(randomString(16)) format Null</query>
```

Comment:
It is actually a strange perf test - https://github.com/ClickHouse/ClickHouse/pull/37894/files#r1290302498
And it is interesting, why client takes 0.758903s to read 1024 bytes at max (at least according to perf tests report)

Reply:
Also maybe the warning about slow on client should be improved, since right now it takes into account time the query executed on the server.

Reply:
indeed, I didn't expect that.

Reply:
regardless of the particular query 1.02 looks a bit too restrictive, cause it is just couple of ms or less for most of the queries 

https://github.com/ClickHouse/ClickHouse/blob/f2621e70aed39f2bf1a061b4820714f84bf7324a/docker/test/performance-comparison/compare.sh#L815-L819